gym/lib/gym/options.rb

```diff
@@ -161,6 +161,11 @@ def self.plain_options
                                      description: "Should an Xcode result bundle be generated in the output directory",
                                      default_value: false,
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :result_bundle_path,
+                                     short_option: "-e",
```

Comment:
any specific motivation for choosing `e`?

Reply:
no real reason, it's the only first letter that was unused, I can remove it if needed.

Reply:
I actually don't know how the short options are selected. @joshdholtz?